zeppelin-web/src/app/helium/helium.html

```diff
@@ -18,32 +18,57 @@
         <h3 class="new_h3">
           Helium
         </h3>
+        <div class="pull-right heliumLearnMore">
+          <a target="_blank"
+             class="helium-repo-btn"
+             ng-href="https://zeppelin.apache.org/helium_packages.html"
```

Comment:
should this be versioned in the future? perhaps newer zeppelin support more helium packages capabilities?